modules/theora/video_stream_theora.h

```diff
@@ -51,16 +51,26 @@ class VideoStreamPlaybackTheora : public VideoStreamPlayback {
 	Point2i size;
 	Rect2i region;
 
+	float *audio_buffer = nullptr;
+	int audio_buffer_size;
+	int audio_ptr_start;
+	int audio_ptr_end;
```

Comment:
Member variables should be default initialized to make sure we don't accidentally access uninitialized memory.